pom.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we keep builder-specific files under https://github.com/slsa-framework/example-package/tree/main/e2e folder. You can create a maven folder

Reply:
Actually, this may be a good idea. Added it in https://github.com/slsa-framework/slsa-github-generator/pull/2538